widget/ui/src/components/FullExpandedQuote/FullExpandedQuote.Skeletons.tsx

```diff
@@ -26,9 +26,7 @@ export function SkeletonHeader() {
         }}>
```

Comment:
The design is always changing. I just checked it and the final version is like this: 
![image](https://github.com/rango-exchange/rango-client/assets/123297941/fa7ece31-65e2-427c-ad92-4f26fb813fbc)